pkg/providers/tinkerbell/hardware/machine.go

```diff
@@ -27,6 +28,74 @@ type Machine struct {
 	BMCUsername  string `csv:"bmc_username, omitempty"`
 	BMCPassword  string `csv:"bmc_password, omitempty"`
 	VLANID       string `csv:"vlan_id, omitempty"`
+
+	// BMCOptions are the options used for Rufio providers.
+	BMCOptions *BMCOptions `csv:"-"`
```

Comment:
How does a user specify the opts?

Reply:
Correct, you cannot set these via the hardware csv. They are all set via env vars only.